lib/DDH/UserPage/Gather.pm

```diff
@@ -167,13 +179,17 @@ sub transform {
                         ( my $login = $developer->{url} ) =~
                             s{https://github.com/(.*)/?}{$1};
 
-                        my %contributor = (
-                            username => $login,
-                            avatar_url => $self->get_avatar($login)
-                        );
+                        $self->add_to_contributors($login, $ia, $ia_id, @contributors);
+                    } elsif ( $developer->{type} && 
+                              $developer->{type} eq 'duck.co' ) {
+
+                            my $complat_user = $self->ddgc->rs('User')->find({ username => $developer->{name} });
+
+                            if ( $complat_user && ( my $gh_id = $complat_user->github_id ) ) {
+                                my $gh_user = $self->ddgc->rs('GitHub::User')->find({ github_id => $gh_id });
```

Comment:
Don't we want to update the database rather than ask the question here? Instead of code complexity to deal with data, fix the data.